src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
The ```FileVersionInfo``` apis to get the FileVersion are not available on .NET Native. This is the best I could find, that will work for all platforms. Let me know if I am missing any other APIs that can be used here.